content/language-agnostic/encyclopedia/code-editor.md

```diff
@@ -0,0 +1,18 @@
+<!--
+---
+Subjects:
+  - "What is a Code Editor?"
+Tags: 
```

Comment:
We're also trying to pull tags as much as possible from the list that's currently in Author (here's [the list](https://author.codecademy.com/browse/tags), also linked to in content-standards.md

Please see if you can use those tags here, but if not (and I suspect this is likely) please alert us to that in the channel as well.